src/core/HttpResponse.ts

```diff
@@ -1,3 +1,4 @@
+import { FetchResponse } from '@mswjs/interceptors'
```

Comment:
`FetchResponse` extends the global `Response` so it's safe to use both in Node.js and in the browser. I hope the root export of Interceptors doesn't load anything Node.js-specific (it mustn't).